Makefile

```diff
@@ -40,6 +45,10 @@ lint: gofmt golint generatenodirty
 .PHONY: cover
 cover:
 	go test -coverprofile=cover.out -coverpkg ./... -v ./...
+	@( echo "Filtering ignored packages..."; \
```

Comment:
optional: instead of filtering from the coverage files, could we break the `go test` up into 2 separate ones:
 - 1 for code that we want coverage on
 - 1 for code that doesn't need coverage

since we don't have tests for the generators, it we don't really need the latter, so it's just a `go test` for packages that we want coverage on (which I think is just `.`)

Reply:
We can generate a `-coverpkg` that doesn't consider the ignored packages.